catalog/core/catalog-core-dynamic-registry/src/main/resources/nitf.xml

```diff
@@ -0,0 +1,321 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
moving this to the nitf transformer where it will be installed as part of the feature